inventory/vagrant/group_vars/all.yml

```diff
@@ -10,3 +10,7 @@ islandora_extra_packages:
   - unzip
   - build-essential
   - vim
+  - cmake
```

Comment:
Since these packages are specific to the grok role. I would install them inside the grok role. 

Maybe inside install.yml using a structure like this: 
https://github.com/Islandora-Devops/claw-playbook/blob/66ef19d323a3c9cc1e1d3e88438d99d3abb0d6e1/roles/internal/openjpeg/tasks/install.yml#L3-L8